### ISSUE TITLE
Error when running logstash setup on aws linux

### ISSUE THREAD

**Issue by wesleymilan** [elastic/logstash#11114]
When running `logstash setup` on AWS Linux the following error appears

```
OpenJDK 64-Bit Server VM warning: Option UseConcMarkSweepGC was deprecated in version 9.0 and will likely be removed in a future release.
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by com.headius.backport9.modules.Modules (file:/usr/share/logstash/logstash-core/lib/jars/jruby-complete-9.2.7.0.jar) to field java.io.FileDescriptor.fd
WARNING: Please consider reporting this to the maintainers of com.headius.backport9.modules.Modules
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
Thread.exclusive is deprecated, use Thread::Mutex
ERROR: Unknown command 'setup'

See: 'bin/logstash --help'
[ERROR] 2019-09-03 17:29:42.751 [main] Logstash - java.lang.IllegalStateException: Logstash stopped processing because of an error: (SystemExit) exit
```

**Comment by maximeaudy** [comment]
The same error for me !

**Comment by nikolamakin** [comment]
what is java version ?

**Comment by maximeaudy** [comment]
the latest

**Comment by kares** [comment]
has been a known issue in JRuby, latest JRuby (9.2.9) is now included in LS 7.5.0
some warnings might still be there but the fatal `java.io.FileDescriptor.fd` is resolved.
